src/NUnitFramework/framework/Internal/Execution/TextMessageWriter.cs

```diff
@@ -21,9 +21,7 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
 using System.Collections;
-using System.Globalization;
```

Comment:
Please amend this change too so that your PR doesn't show up in the history of this file.

Reply:
Oh :roll_eyes: , i forgot that, my used plug-in, **Power Commands for VS** automatic remove unused **using** on save